spec/mocks/MockCompiler.coffee

```diff
@@ -0,0 +1,57 @@
+$ = require('jquery')
+Backbone = require('backbone')
+
+MockCompiler = Backbone.Model.extend(
+  load: (templateName, onCompletion) ->
+    html = @getHTMLForTemplateName(templateName)
+    onCompletion(html)
+
+  appendToContainer: (container, template) ->
+    $(container).html(template)
+
+  getHTMLForTemplateName: (templateName) ->
```

Comment:
Would you have to change this markup if that of the views changes? If so, this is a form of duplication. How can you eliminate it?